src/Core/FSpot.Core.UnitTest/BrowsablePointerTests.cs

```diff
@@ -0,0 +1,171 @@
+//
+//  BrowsablePointerTests.cs
+//
+// Author:
+//   Daniel Köb <daniel.koeb@peony.at>
+//
+// Permission is hereby granted, free of charge, to any person obtaining
+// a copy of this software and associated documentation files (the
+// "Software"), to deal in the Software without restriction, including
+// without limitation the rights to use, copy, modify, merge, publish,
+// distribute, sublicense, and/or sell copies of the Software, and to
+// permit persons to whom the Software is furnished to do so, subject to
+// the following conditions:
+//
+// The above copyright notice and this permission notice shall be
+// included in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED AS IS, WITHOUT WARRANTY OF ANY KIND,
+// EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+// LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+// WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+
+using NUnit.Framework;
+using System.Linq;
+using Hyena;
+using System.Collections.Generic;
+
+namespace FSpot.Core.UnitTest
+{
+    [TestFixture]
+    public class BrowsablePointerTests
+    {
+        private IPhoto photo1 = new FilePhoto(new SafeUri("/1"));
+        private IPhoto photo2 = new FilePhoto(new SafeUri("/2"));
+        private IPhoto photo3 = new FilePhoto(new SafeUri("/3"));
```

Comment:
I think it is in the next version. If you want to be adventurous you could build from source. MonoDevelop 6.0 which is actually roslyn based. So, you'll be using the same backend on MonoDevelop on Linux as you would be in VS 2015 on Windows. Build from source is fairly straightforward.

Another feature that is pretty cool is the unit test integrate. It adds a dot next to each test in a test file. It has a context menu too that lets you Run, Debug, etc a specific test.

There are several other things that I change. You just have to dig through the preferences dialog. :)